lib/iris/plot.py

```diff
@@ -403,8 +405,14 @@ def _draw_2d_from_points(draw_method_name, arg_func, cube, *args, **kwargs):
 
 def _fixup_dates(coord, values):
     if coord.units.calendar is not None and values.ndim == 1:
-        r = [datetime.datetime(*(coord.units.num2date(val).timetuple()[0:6]))
-             for val in values]
+        num2date = coord.units.num2date
+        if coord.units.calendar == 'gregorian':
```

Comment:
is this code path necessary? can netcdftime handle the gregorian calendar as well?